tests/unit/mock_data/counts/test_workflow_status_failed_helloword_metadata.json

```diff
@@ -0,0 +1 @@
+	HelloWorld.HelloWorldTask	0 Running, 0 Done, 0 Preempted, 1 Failed
```

Comment:
This doesn't look like a valid json file. Can you check?

Reply:
Nope, meant to be a txt file.